browser/src/layer/vector/SVGGroup.js

```diff
@@ -78,6 +80,31 @@ L.SVGGroup = L.Layer.extend({
 		var point = this._map.latLngToLayerPoint(this._bounds.getNorthWest());
 		svgLastChild.setAttribute('x', point.x);
 		svgLastChild.setAttribute('y', point.y);
+
+		var video = svgLastChild.getElementsByTagName('video')[0];
+		var source = svgLastChild.getElementsByTagName('source')[0];
+		this.checkVideoSupport(video, source);
+	},
+
+	checkVideoSupport: function(video,source) {
+		var that = this;
+		var videoWarning = _('Document contains uncompatible video');
```

Comment:
Shouldn't we have this in one place only? It's duplicated here and in Map.SlideShow.js.